superset-frontend/src/explore/components/controls/DateFilterControl/components/CustomFrame.tsx

```diff
@@ -132,6 +139,7 @@ export function CustomFrame(props: FrameComponentProps) {
                   onChange('sinceDatetime', datetime.format(MOMENT_FORMAT))
                 }
                 allowClear={false}
+                locale={locale}
```

Comment:
Thanks for the contribution! Could we make translation more generic? For example, we can define a map constant that mapping from `bootstrapData.common.local` to the `antd locale`. feel free to leave messages if you need help.

Reply:
Thanks for the review, I agree it will be better with mapping.
I just wonder where I can put this map and I don't really see how I can integrate an import in a map

Reply:
Hey @zhaoyongjie, I change the process to choose dynamicly the locale.
I use a list instead of import all local we need
Just for french translation, there is a bug in antd. We can't distinguish the fr from france and belgium properly so I add a if for now